src/models/network/NetworkAppController.js

```diff
@@ -171,13 +171,18 @@ export default class NetworkAppController extends NetworkBaseController {
 
     const stdlibName = this.packageFile.stdlibName;
     log.info(`Connecting to public deployment of ${stdlibName} in ${this.networkFile.network}`);
-    const stdlibAddress = Stdlib.fetch(stdlibName, this.packageFile.stdlibVersion, this.networkFile.network);
+    const { address: stdlibAddress, version: stdlibVersion } = Stdlib.fetch(stdlibName, this.packageFile.stdlibVersion, this.networkFile.network);
 
-    if(stdlibAddress !== currentStdlibAddress) {
+    if (stdlibAddress !== currentStdlibAddress) {
       await this.app.setStdlib(stdlibAddress);
-      this.networkFile.stdlib = { address: stdlibAddress, ... this.packageFile.stdlib };
+      this.networkFile.stdlib = { 
+         ... this.packageFile.stdlib,
+         address: stdlibAddress, 
+         version: stdlibVersion
```

Comment:
which attributes are we writing from `packageFile.stdlib`? I think it is just the `name`, right? In case I'm not wrong, I think it would be clearer making it explicit, do you agree?

Reply:
It's also the `customDeployment` flag. I'd rather ensure all info is copied.